### DISCUSSION TITLE
PR status is not synchronized.

### DISCUSSION THREAD

**Discussion by psbss** [community/community#42782] [Pull Requests] [answered]
### Select Topic Area

Bug

### Body

Hello everyone 🙌.

I have a problem with pull request body text.

If you write the number of another PR in the body and save it, GitHub will display it as shown in the image below.
I have recently been struggling with a problem where I changed the status of this PR to Merged and the status is not updated.
This is not a browser cache issue, but a GitHub server issue.

This problem can be solved by editing the text.
It worked fine until November 2022.

👇 Wrong status like this
![20221229_010705](https://user-images.githubusercontent.com/24839924/209840826-301ff774-16dd-46f7-858a-b5bfea8f7db7.png)

👇 correct status like this
![20221229_011454](https://user-images.githubusercontent.com/24839924/209841079-76f3beec-34f5-43b8-ab26-20f21111984c.png)

**Comment by MikeMcC399** [comment]
I relied quite a lot on the feature of having the status of a PR appear in an issue or another PR by adding it to a list. 

Reading the help on https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/about-task-lists it sounds like this was partially a beta feature. Is it now necessary to add `[ ]` to get the status to update automatically?

**Comment by psbss** [comment, marked as answer]
solved
